### PR TITLE
Run all ClinSeq related commands through perltidy

### PR DESCRIPTION
**Pull request by susannasiebert**
There will probably be some merge conflicts once #1318 gets merged.

### CODE REVIEW

**Comment by jasonwalker80** [comment]
+1, once tests pass.

File: lib/perl/Genome/Model/ClinSeq/Command/AnnotateGenesByCategory.t [hunk 1]
```diff
@@ -77,13 +86,13 @@ ok(-e $log_file2, "Wrote message file from annotate-genes-by-category to a log f
...
-};
+    or do {
+    diag("expected: $expected_output_dir\nactual: $temp_dir\n");
+    diag("differences are:");
+    diag(@diff);
+    my $diff_line_count = scalar(@diff);
+    print "\n\nFound $diff_line_count differing lines\n\n";
+    Genome::Sys->shellcmd(cmd => "rm -fr /tmp/last-annotate-genes-by-category");
+    Genome::Sys->shellcmd(cmd => "mv $temp_dir /tmp/last-annotate-genes-by-category");
+    };
```
Comment (tmooney) [inline lib/perl/Genome/Model/ClinSeq/Command/AnnotateGenesByCategory.t]: I wonder why perltidy has the contents of this block and the closing `}` at the same indentation.

**Comment by tmooney** [comment]
+1, even though I think the `do` blocks look funny!

File: lib/perl/Genome/Model/ClinSeq/Command/AnnotateGenesByCategory.t (hunk 1 above)
Comment (susannasiebert) [inline lib/perl/Genome/Model/ClinSeq/Command/AnnotateGenesByCategory.t]: I'm not sure but I bet it has something to do with the `or`. I will take a closer look and make it look nicer.

**Pull request merged**